blocks/README.md

```diff
@@ -219,3 +219,35 @@ function edit( props ) {
 	} );
 }
 ```
+
+## Matchers
+
+Given the block's HTML content as a string, attribute matchers extract the block attributes values into an Object. The matchers are made available throught the `wp.blocks.query` variable as superset of [HPQ](http://github.com/aduth/hpq)'s API.
+
+`attr( selector: ?string, name: string ): Function`
+
+Generates a matcher function, returning an attribute by name if the attribute exists. If no selector is passed, returns attribute of the root element.
```

Comment:
This is a bit confusing. Can we explain it simpler? If an attributes exists _where_? Maybe we can change the phrasing to be more about: "Use `attr` when you want to extract y from x`.